### ISSUE TITLE
Github pa11y workflow task sometimes fails

### ISSUE THREAD

**Issue by chriswilty** [ScottLogic/blog#290]
Sometimes our accessibility workflow task fails with error "no usable sandbox":
https://github.com/ScottLogic/blog/pull/287/checks#annotation:13:9

This seems to be due to Ubuntu linux restricting unprivileged namespace access:
https://chromium.googlesource.com/chromium/src/+/main/docs/security/apparmor-userns-restrictions.md

Going on the above (see section "Why does this only affect developer builds?"), using `stable` as release version rather than `latest` (which is the default) in the setup-chrome workflow config should fix the problem:
https://github.com/browser-actions/setup-chrome

While we are fixing that, we should also make use of the output from this action when invoking chrome in the subsequent "Run pa11y-ci" step. See https://github.com/browser-actions/setup-chrome?tab=readme-ov-file#installed-path
